app/parse.py

```diff
@@ -8,5 +11,26 @@ class Course:
     duration: str
 
 
+def parse_single_course(course_element: Tag) -> Course:
+    name = course_element.select_one("h3").text.strip()
+    short_description = course_element.select_one(
+        ".typography_landingTextMain__Rc8BD"
+    ).text.strip()
+    duration = course_element.select(
+        ".typography_landingTextMain__Rc8BD span"
+    )[-1].text.strip()
```

Comment:
Ensure that the selector '.typography_landingTextMain__Rc8BD span' correctly targets the intended element for the duration. If the structure of the HTML changes, this could lead to errors or incorrect data extraction.